Package.swift

```diff
@@ -0,0 +1,34 @@
+// swift-tools-version:5.2
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "ZetaPush",
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "ZetaPush",
+            targets: ["ZetaPush"]),
```

Comment:
lol ! you named it ZetaPush. But long time ago and I repeated it last week that this lib is ZPNetwork or ZetaPushNetwork for Client. ZetaPush is the entire plateform or the original society. It seems strange to name this sdk zetapush.

Reply:
In addition to this to understand the all context, this network sdk works with the generated files comes from backend that we called zpAPI to dissociate this two instancies.